jekyll/_cci2/ios-migrating-from-1-2.md

```diff
@@ -0,0 +1,406 @@
+---
+layout: classic-docs
+title: "Migrating your iOS project from 1.0 to 2.0"
+short-title: "Migrating your iOS project from 1.0 to 2.0"
+description: "How to migrate your iOS project from CircleCI 1.0 to 2.0"
+categories: [migration]
+order: 16
+---
+
+This document will give you the guidelines for migrating your iOS
+project from CircleCI 1.0 to 2.0.
+
+## Overview
+
+With the release of CircleCI 2.0 for macOS, your iOS projects can now
+benefit from the improvements in our 2.0 platform, including:
+
+* Workflows: Orchestrate jobs and steps with great flexibility using a
+  simple set of new keys in your configuration.
+
+* Advanced caching: Speed up builds by caching files from run to run
+  using keys that are easy to control with granular caching options for
+  cache save and restore points throughout your jobs. Cache any files
+  from run to run using keys you can control.
+
+## Full example of the configuration
+
+This sample configuration file should work for most iOS projects on
+CircleCI 2.0:
+
+```
+# .circleci/config.yml
+
+# Specify the config version - version 2 is latest.
+version: 2
+
+# Define the jobs for the current project.
+jobs:
+  build-and-test:
+
+    # Specify the Xcode version to use.
+    macos:
+      xcode:
+        version: "8.3.3"
+
+    # Define the steps required to build the project.
+    steps:
+
+      # Get the code from the VCS provider.
+      - checkout
+
+      # Restore all cached files.
+      - restore_cache:
```

Comment:
I'd suggest that we remove Gemfiles and caching from the initial example, and add a second example in the docs at a later date on using Gemfiles to pin tool versions. This will make your (excellent) example shorter.